src/main/groovy/org/boozallen/plugins/jte/utils/FileSystemWrapper.groovy

```diff
@@ -42,6 +42,10 @@ class FileSystemWrapper {
     SCMFileSystem fs
     String scmKey
 
+    static FileSystemWrapper create(SCM scm){
```

Comment:
This is fine for now - since the next thing we'll do is make this more explicit via something like ``FileSystemWrapper.createFromSCM(scm)`` and ``FileSystemWrapper.createFromCurrentJob()`` right?

Reply:
correct, I will work on that re-work of the FileSystemWrapper factory methods next